### PR TITLE
Fixing data generation to re-fetch participant before updating status; updating metrics pipeline to run at 4 EST.

### PR DESCRIPTION
**Pull request by danqrodney**
Changing metrics pipeline to run at 4 AM, after biobank samples pipeline should have finished.

### CODE REVIEW

File: rest-api/cron.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 cron:
 - description: Daily metrics
   url: /offline/MetricsRecalculate
-  schedule: every day 01:00
+  schedule: every day 04:00
```
Comment (markfickett) [inline rest-api/cron.yaml]: Do you mean for this to be in the same PR? Would you mind either calling it out in the PR title or splitting it up? I'd like it to show up in release notes.

File: rest-api/data_gen/fake_participant_generator.py [hunk 2]
```diff
@@ -489,13 +489,17 @@ def _submit_hpo_changes(self, participant_response, participant_id, consent_time
 
   def _submit_status_changes(self, participant_response, participant_id, last_request_time):
```
Comment (markfickett) [inline rest-api/data_gen/fake_participant_generator.py]: The participant_response parameter can be removed I think.

Comment (markfickett) [inline rest-api/data_gen/fake_participant_generator.py]: It looks like this is called immediately after the response is fetched; why does it need to be re-fetched?

Comment (markfickett) [reply rest-api/data_gen/fake_participant_generator.py]: Chatted offline -- adding a comment will be useful, but makes sense now.

File: rest-api/cron.yaml (hunk 1 above)
Comment (danqrodney) [reply rest-api/cron.yaml]: Added to PR title.

File: rest-api/data_gen/fake_participant_generator.py (hunk 2 above)
Comment (danqrodney) [reply rest-api/data_gen/fake_participant_generator.py]: Done.

Comment (danqrodney) [reply rest-api/data_gen/fake_participant_generator.py]: Added comments.

**Pull request merged**